Mantle/MTLModel.m

```diff
@@ -283,7 +323,7 @@ - (BOOL)validate:(NSError **)error {
 #pragma mark NSCopying
 
 - (instancetype)copyWithZone:(NSZone *)zone {
-	return [[self.class allocWithZone:zone] initWithDictionary:self.dictionaryValue error:NULL];
+	return [[self.class allocWithZone:zone] initWithValidatedDictionary:self.dictionaryValue error:NULL];
```

Comment:
We could totally do that. I was thinking the exceptions might be useful to keep, but, you're right about there not being any way to bubble them up. Will update the PR.